storage/tests/unit/test_blob.py

```diff
@@ -1009,9 +1013,11 @@ def _initiate_resumable_helper(
         data = b'hello hallo halo hi-low'
         stream = io.BytesIO(data)
         content_type = u'text/plain'
+        predefined_acl = 'private'
```

Comment:
Shouldn't `_initiate_resumable_helper` take `predefined_acl` as an argument, like `_do_multipart_helper`.  We need to continue to exercise the codepath *without* the value passed in, too.